project-management/agile-opensource-workflow.md

```diff
@@ -7,6 +7,10 @@ This document describes a workflow that combines participatory, open source and
 *There needs to be something here that provides a little background about why we need a whole new way of working.*
 *Something along the lines of the difference between continual and continuous, and the difference between people who are paid to work on the project vs people who are volunteering their time - both members of the autistic community and volunteer developers.*
 
+This is a project funded by and supported by the Autistic community and therefore every stage of the project requires complete transparency and community feedback. A new way of working needs to be developed to incorporate Alan Turing and Fujitsu to work effectively together. Fujitsu work in an Agile way to develop the platform efficiently.
```

Comment:
```suggestion
This is a project funded by and supported by the Autistic community and therefore every stage of the project requires complete transparency and community feedback.
A new way of working needs to be developed to incorporate Alan Turing and Fujitsu to work effectively together.
All parties are committed to working in an Agile (and agile) way to develop the platform efficiently.
```

Reply:
incorporating @JamesSCTJ's suggestion (and new line formatting).

Reply:
I'll have to commit this separately as I don't have permission to commit to @EllenDevereux's fork.

Reply:
Done!